roles/attached-volumes/tasks/set_up_volume.yml

```diff
@@ -1,11 +1,11 @@
 ---
 - block:
-    - name: Detect attached volume
-      parted:
-        device: "{{ attached_volume.device }}"
-        unit: GiB
-      register: volume_info
     - block:
+      - name: Detect attached volume
+        parted:
+          device: "{{ attached_volume.device }}{{ attached_volume.partition }}"
```

Comment:
{{ attached_volume.partition|d('') }}

Reply:
otherwise this could complain about an undefined variable for things without partition set